scan_topic : /otsu_filter/filteredCloud
detected_clusters_topic: /radar_detected_clusters
detected_clusters_marker_topic: /radar_detected_clusters_marker

#fixed_frame : radar_right
fixed_frame : summit_xl_odom
#fixed_frame : summit_xl_base_link
#fixed_frame: summit_xl_front_laser_link

classifier_type: opencv_random_forest
model_file : $(find radar_squat_leg_tracker)/config/trained_model/TrainBoth/opencv_random_forest_3.yaml

publish_background : false # whether publish non-human clusters (true for static robot, false for moving robot)
visualize_contour : false # whether visualize cluster contour in rviz
use_scan_header_stamp_for_tfs : false

cluster_dist_euclid : 0.3
min_points_per_cluster : 5

detection_threshold : -1.0 # publish clusters with confidence level above the threshold
max_detected_clusters : -1 # maximum number of published clusters
max_detect_distance : 8.0 # maximum distance of published clusters to sensor
